app/javascript/App/components/FilterPanel/index.js

```diff
@@ -30,16 +30,18 @@ class FilterPanel extends Component {
     const pProps = { ...props, filterData: normalizeFilterData(filterData) }
     const { applyFilters, resetFilters } = this
     return (
-      <Layout {...{ ...props, applyFilters, resetFilters }}>
+      <Layout {...{ ...props, applyFilters, resetFilters }} newsFeedStyle>
         {filterList.map((filter, index) => {
           if (filter.get('key') === 'coins') return null // Temp fix for hiding coins
+          if (filter.get('key') === 'keywords') return null
```

Comment:
We definitely want to hide this? I seem to remember hearing that it would be nice to have

Reply:
it's great that you remember hearing something about how the requirements may have evolved. i think we should move to be adamant about having a single source of truth. that could very well be the design comps. 

this is non-existent in the comps so i took it out. but it's just one line to put it back, when we need it!